src/core/src/bootstrap/Constants.py

```diff
@@ -185,6 +184,14 @@ class RebootStatus(EnumBackport):
         STARTED = "Started"
         COMPLETED = "Completed"
         FAILED = "Failed"
+    
+    # Enum for VM Type
+    class VMType(EnumBackport):
```

Comment:
VMCloudType;
also if you're having Unknown in the enum, the standard practice is to have it first in the enum